src/types/index.d.ts

```diff
@@ -1,3 +1,11 @@
+import { Menu } from '@/types/dataTypes/drupal/menu'
+
+export global {
```

Comment:
We removed this intentionally; couldn't figure out why it was there.

Reply:
copy. will remove